src/game-status/adiMalka.js

```diff
@@ -0,0 +1,28 @@
+const optionToGuess = 4;
+
+/**
+ * thr function get array of strung - colors and return random array of new options
+ * @param {[string]} colorOptions
+ * @return {[string]} colorsToGuess
+ */
+function getColorsToGuess(colorOptions){
+    return new Array(optionToGuess).fill(0)
+        .map(() => colorOptions[Math.floor(Math.random()*colorOptions.length)]);
+}
+
+/**
+ * thr function get array of strung - colors and return random array of new options
+ * @param {string} userInput
+ * @param {[string]} colorOptions
+ * @return {[string]} colorsToGuess
+ */
+function convertUserGuessToArray(userInput,colorOptions ){
+    const numbersInput = userInput.trim().replace(/\s+/g, ' ').split(' ');
+
+    if(numbersInput.length !== optionToGuess ||
```

Comment:
create a function that validates the input. even create a utils folder and a file for it.